### DISCUSSION TITLE
希望提供给用户输出配置文件以供用户备份配置的按钮

### DISCUSSION THREAD

**Discussion by nigeder** [shuax/MouseInc.Settings#97] [Ideas]
目前的重置给功能因为是全局重置，因为一两个设置的调试而重置全部恢复起来太麻烦，建议能有配置文件备份的功能，谢谢。

**Comment by shuax** [comment]
自己备份mouseinc.json就是了。

**Comment by zhjcdw** [comment]
不用专门的备份选项吧，用记事本就可以打开配置文件，按照语法规则，想怎么改都可以。我就是看中这一点，才选择mouseinc
